go.mod

```diff
@@ -7,10 +7,13 @@ require (
 	github.com/cenkalti/backoff/v4 v4.1.0
 	github.com/client9/misspell v0.3.4
 	github.com/crewjam/saml v0.4.5
+	github.com/google/go-cmp v0.5.4 // indirect
```

Comment:
No need to include indirect packages in go mod

Reply:
@bogdanprodan-okta   Updated the go mod